deploy/crds/org_v1_che_crd.yaml

```diff
@@ -446,10 +442,9 @@ spec:
                     its CA cert to be able to request it. This is disabled by default.
                   type: string
                 tlsSupport:
-                  description: 'Instructs the operator to deploy Che in TLS mode,
-                    ie with TLS routes or ingresses. This is disabled by default.
-                    WARNING: Enabling TLS might require enabling the `selfSignedCert`
-                    field also in some cases.'
+                  description: Instructs the operator to deploy Che in TLS mode. This
```

Comment:
Move `Deprecated` to the beginning of the description

Reply:
ok